### DISCUSSION TITLE
3 month Construction Timelaps. I'm having difficulties

### DISCUSSION THREAD

**Discussion by Shosho80** [gopro/labs#925] [Q&A]
Hello, I need to do a 3 month timelapse of a construction in a closed hall. i have 6 gopro 12 cameras which i will put them at 9 meters high. I planned to run the cameras on:
1. SuperPhoto/Linear
2. in 10 minute intervals to turn on to take a photo and then turn off. to do so from Monday to Friday. from 07:00 to 19:00.
3. to download the photos to the cloud space of gopro.

I've run into a big challenge and I'm having trouble with the Labs code language. Please help me with a code to put in my cameras. Construction starts in a week. Thank you very much friends

**Comment by dnewman-gpsw** [comment]
If you read the forum, you should know if you want reliability, avoid the upload.  Just test the once every few weeks.

**Comment by Espusen** [comment]
It's the wholy grail that I think we are very close to!
My testing has been unreliable sadly with upload.
The good new is that the codes are now very reliable for scheduling the cameras with NO upload!! 

Best bet I would say is to run a standard Construction code as you describe, 0700-1900, 10 min interval but include weekends to reduce complexity. Also no upload.
 Just check on them every so often.
